### DISCUSSION TITLE
Change system font to a mono font

### DISCUSSION THREAD

**Discussion by acester822** [CosmosOS/Cosmos#3110] [Q&A] [answered]
I have a thing for VictorMono and I use it everywhere on everything. The issue I am having is that it is not popping up as an option for system font, only monospace fonts. Is there a way I can set it for ALL fonts?

**Comment by Samma2009** [comment]
> I have a thing for VictorMono and I use it everywhere on everything. The issue I am having is that it is not popping up as an option for system font, only monospace fonts. Is there a way I can set it for ALL fonts?

What does this mean, where Is the font missing from

**Reply by acester822** [reply]
From system fonts.....in cosmos settings

![screenshot-2024-12-08-00-44-24](https://github.com/user-attachments/assets/9f69336d-f5d7-454d-b40f-0d664c7ca4c8)

**Reply by zarlo** [reply]
i think you are looking for the cosmic desktop from https://github.com/pop-os

**Reply by acester822** [reply]
> i think you are looking for the cosmic desktop from https://github.com/pop-os

I am on the latest alpha cosmic on pop-os. All I am trying to do is change the system font to Victor Mono

**Reply by zarlo** [reply, marked as answer]
you are on the wrong git repo, this is for the C# Open Source Managed Operating System

**Reply by acester822** [reply]
> you are on the wrong git repo, this is for the C# Open Source Managed Operating System

Welp. No idea how that happened lmao. Thanks guys, turns out I'm the idiot lol.
